### PR TITLE
gcp-lts-bom July 2022

### PR DESCRIPTION
**Pull request by suztomo**

### CODE REVIEW

**Comment by suztomo** [comment]
`mvn test -Dtest='BomContentTest#testLtsBom'` fails due to the appengine-api-1.0-sdk version. This is a provided-scope dependency and Gradle or Maven doesn't use the dependency when they run dependency tree.

File: boms/cloud-lts-bom/pom.xml [hunk 1]
```diff
@@ -80,8 +80,8 @@
     <google-cloud-tasks.version>2.1.11</google-cloud-tasks.version>
     <bigtable-hbase-beam.version>2.1.1</bigtable-hbase-beam.version>
     <google-cloud-bigtable.version>2.6.2</google-cloud-bigtable.version>
-    <google-cloud-pubsub.version>1.116.4</google-cloud-pubsub.version>
-    <proto-google-cloud-pubsub-v1.version>1.98.4</proto-google-cloud-pubsub-v1.version>
+    <google-cloud-pubsub.version>1.117.0</google-cloud-pubsub.version>
```
Comment (suztomo) [inline boms/cloud-lts-bom/pom.xml:83]: As per discussion with Mike M.

**Review by claudio4j — APPROVED** [review]

**Comment by claudio4j** [comment]
Hi, great these versions were updated in the bom. Excuse me to ask, when is this planned to be merged and a new lts bom version released ? Thanks.

**Comment by suztomo** [comment]
@claudio4j This is for Google internal stuff and not for customers. Please use the Libraries BOM (the latest is today's release `com.google.cloud:libraries-bom:25.3.0`) instead. https://cloud.google.com/java/docs/bom

**Comment by claudio4j** [comment]
I am mostly interested in this libraries, which are updated in this bom.
```
<google-http-client.version>1.41.8</google-http-client.version>
<google-oauth-client.version>1.33.3</google-oauth-client.version>
<google-api-client.version>1.34.1</google-api-client.version>
```
I see these libraries are not in the `com.google.cloud:libraries-bom`, is there other bom for me to import ? Sorry to use this PR to ask this.

**Comment by suztomo** [comment]
Let's continue in https://github.com/GoogleCloudPlatform/cloud-opensource-java/issues/2307

File: boms/cloud-lts-bom/pom.xml [hunk 2]
```diff
@@ -49,50 +49,51 @@
...
+    <proto-google-cloud-pubsub-v1.version>1.99.0</proto-google-cloud-pubsub-v1.version>
+    <google-cloud-datastore.version>2.2.10</google-cloud-datastore.version>
+    <google-cloud-service-usage.version>2.2.7</google-cloud-service-usage.version>
+    <google-cloud-container.version>2.3.7</google-cloud-container.version>
+    <google-cloud-orchestration-airflow.version>1.1.7</google-cloud-orchestration-airflow.version>
+    <google-cloud-resourcemanager.version>1.2.11</google-cloud-resourcemanager.version>
+    <google-cloud-redis.version>2.4.1</google-cloud-redis.version>
+    <google-cloud-logging.version>3.7.5</google-cloud-logging.version>
+    <google-cloud-secretmanager.version>2.1.6</google-cloud-secretmanager.version>
```
Comment (suztomo) [inline boms/cloud-lts-bom/pom.xml:92]: This is new addition.

**Review by blakeli0 — APPROVED** [review]

**Pull request merged**